frontend/src/utils/scheduler/scheduler.ts

```diff
@@ -175,8 +175,8 @@ export const serializeSchedule = (schedule: Schedule, semester: Semester): Backe
  * Generates iCal file string.
  */
 export function scheduleToICal(schedule: Schedule, semester: Semester): string {
-	const SEMESTER_START = new Date(2023, 0, 17);
-	const LAST_COURSE_DAY = new Date(2023, 4, 6);
+	const SEMESTER_START = new Date(2023, 7, 23);
+	const LAST_COURSE_DAY = new Date(2023, 11, 8);
```

Comment:
As noted before, `LAST_COURSE_DAY` should be `new Date(2023, 11, 1)`.

Reply:
Oh yeah you're right, thanks for catching this.